de.dashup.shared/src/main/java/de/dashup/shared/DatabaseModels/DatabaseUser.java

```diff
@@ -0,0 +1,95 @@
+package de.dashup.shared.DatabaseModels;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class DatabaseUser extends DatabaseObject {
+
+    private Integer id;
+    private String email;
+    private String username;
+    private String name;
+    private String surname;
+    private Date birthdate;
+    private String company;
+    private String biography;
+    private String password;
+    private String salt;
+    @SerializedName("background_image")
+    private String backgroundImage;
```

Comment:
Why not leaving the settings in the settings? This causes everything that I have implemented in #72 to fail. Furthermore we decided to separate the settings from the user.

Reply:
See one of the comments above (normalization, implementing performance thoughts, ...)